lib/options.js

```diff
@@ -69,6 +69,12 @@ Version ${ helpers.getVersion() }
       type: 'Boolean',
       description: 'AMD and CommonJS'
     },
+    {
+      option: 'es6Modular',
+      alias: 'M',
```

Comment:
please remove the alias. I think having two aliases with `M` and `m` could be confusing for the end users. `--esm` should be enough